File: __tests__/fixtures/output/python/python3/jsonObj-multiline.py
import http.client

conn = http.client.HTTPSConnection("httpbin.org")

payload = "{\n  \"foo\": \"bar\"\n}"

headers = { 'content-type': "application/json" }

conn.request("POST", "/anything", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: src/targets/javascript/jquery/fixtures/jsonObj-multiline.js
const settings = {
  async: true,
  crossDomain: true,
  url: 'https://httpbin.org/anything',
  method: 'POST',
  headers: {
    'content-type': 'application/json'
  },
  processData: false,
  data: '{\n  "foo": "bar"\n}'
};

$.ajax(settings).done(res => {
  console.log(res);
});